packages/connect-web/conformance/client.ts

```diff
@@ -34,11 +33,13 @@ const { values: flags } = parseArgs({
   options: {
     browser: { type: "string", default: "chrome" },
     headless: { type: "boolean" },
+    openBrowser: { type: "boolean" },
```

Comment:
Instead of opting in to run headless with `--headless`, you now opt in to run with UI with `--openBrowser`. This allows us to re-use the npm scripts. It's explained in packages/connect-conformance/README.md